ui/org.eclipse.pde.ui/src/org/eclipse/pde/ui/launcher/PluginsTab.java

```diff
@@ -156,6 +157,17 @@ public void createControl(Composite parent) {
 
 	@Override
 	public void initializeFrom(ILaunchConfiguration configuration) {
+		// Long-running initialization happens on first activation of this tab
+		System.out.println("PluginsTab.initializeFrom()");
```

Comment:
Please remove all debug println from PR

Reply:
Done in dc7ca510a2bf0f3d14a7c7600905e6bc1ab36b93